data/searchers/searchers first contact.txt

```diff
@@ -168,14 +168,14 @@ mission "Talk to Quarg about Searchers"
 				`	(Yes.)`
 				`	(Not right now.)`
 					defer
-			`	It doesn't take long for you to find one of the Quarg who's willing to speak with you. "Greetings, human," says the Quarg. "You seem as if you wish to say something. What is it?" When you tell the Quarg about the Searchers and the ruined Wardragon they found in their space, the Quarg seems saddened. "Ah, yes, the Searchers, who call themselves the '|^\-']. Such a promising species, they once were. Then, the Pug foolishly decided to box them into half their proper number of systems and let them stagnate. They said that sooner or later, they would expand beyond those borders. Naturally, the Quarg disagreed. We saw that it was far better to let them grow, but the Pug did not listen. When we sent a Wardragon to keep an eye on the system, it vanished. That is what the Searchers found, before we could remove it."`
+			`	It doesn't take long for you to find one of the Quarg who's willing to speak with you. "Greetings, human," says the Quarg. "You seem as if you wish to say something. What is it?" When you tell the Quarg about the Searchers and the ruined Wardragon they found in their space, the Quarg seems saddened. "Ah, yes, the Searchers, who call themselves the '|^\-']. Such a promising species, they once were. Then, the Pug foolishly decided to box them into half their proper number of systems and let them stagnate. They said that sooner or later, they would expand beyond those borders. Naturally, we Quarg disagreed. We saw that it was far better to let them grow, but the Pug did not listen. When we sent a Wardragon to keep an eye on the system, it vanished. That is what the Searchers found, before we could remove it."`
```

Comment:
Hmm... I probably need to completely rewrite this mission anyways, but for now this is a good change.